korman/properties/modifiers/region.py

```diff
@@ -203,6 +203,237 @@ def requires_actor(self):
+                ("kFlagReverbScale", "Reverb Scale", "Reverberation level"),
+                ("kFlagReverbDelayScale", "Reverb Delay Scale", "Late reverberation delay time"),
+                ("kFlagEchoTimeScale", "Echo Time Scale", "Echo time"),
+                ("kFlagModulationTimeScale", "Modulation Time Scale", "Modulation time"),
+                ("kFlagDecayHFLimit", "Decay HF Limit", "Limit unnaturally long decay times of high-frequency sounds by forcing a limit to the decay time to be calculated from the Air Absorption HF value")]
+
+class PlasmaReverbRegion(PlasmaModifierProperties):
+    pl_id = "reverb"
+    pl_depends = {"softvolume"}
+
+    bl_category = "Region"
+    bl_label = "Sound Reverb Region"
+    bl_description = "Sound Reverberation (EAX) Region"
+    bl_icon = "IPO_ELASTIC"
+
+    preset = EnumProperty(name="Environment Preset",
+                          description="The type of audio environment to simulate",
+                          items=[("GENERIC", "Generic", "A generic-sounding environment with light reverberation"),
+                                 ("PADDEDCELL", "Padded cell", ""),
+                                 ("ROOM", "Room", ""),
+                                 ("BATHROOM", "Bathroom", ""),
+                                 ("LIVINGROOM", "Living room", ""),
+                                 ("STONEROOM", "Stone room", ""),
+                                 ("AUDITORIUM", "Auditorium", ""),
+                                 ("CONCERTHALL", "Concert Hall", ""),
+                                 ("CAVE", "Cave", ""),
+                                 ("ARENA", "Arena", ""),
+                                 ("HANGAR", "Hangar", ""),
+                                 ("CARPETTEDHALLWAY", "Carpetted hallway", ""),
+                                 ("HALLWAY", "Hallway", ""),
+                                 ("STONECORRIDOR", "Stone corridor", ""),
+                                 ("ALLEY", "Alley", ""),
+                                 ("FOREST", "Forest", ""),
+                                 ("CITY", "City", ""),
+                                 ("MOUNTAINS", "Mountains", ""),
+                                 ("QUARRY", "Quarry", ""),
+                                 ("PLAIN", "Plain", ""),
+                                 ("PARKINGLOT", "Parking lot", ""),
+                                 ("SEWERPIPE", "Sewer pipe", ""),
+                                 ("UNDERWATER", "Underwater", ""),
+                                 ("DRUGGED", "Drugged", ""),
+                                 ("DIZZY", "Drizzy", ""),
+                                 ("PSYCHOTIC", "Psychotic", ""),
+                                 ("MORE", "More choices...", ""),
+                                 ("CUSTOM", "Custom", "Setup your own environment")],
+                          default="GENERIC",
+                          options=set())
+
+    # Thikk list for annoying users.
```

Comment:
For correct spelling:
```suggestion
    # Thick list for annoying users.
```

For innuendo:
```suggestion
    # Thicc list for annoying users.
```

😛

Reply:
Ah, the silly version is the one I had in mind. Assuming you don't mind :P